trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -205,6 +210,56 @@ const sendManualBankDecisionEmail = async (amendmentVariables) => {
   }
 };
 
+/**
+ * Ascertain whether the requested amendment
+ * have been declined or not.
+ * @param {Object} amendment Amendment object
+ * @returns {Boolean} Whether both the amendments decision has been declined by the underwriter.
+ */
+const amendmentDeclined = (amendment) => {
+  const { changeFacilityValue, changeCoverEndDate } = amendment;
+  const { value, coverEndDate } = amendment.ukefDecision;
+
+  // Ensure not all of the amendment requests are declined
+
+  // Dual amendment request
+  if (changeFacilityValue && changeCoverEndDate) {
+    return value === UNDERWRITER_MANAGER_DECISIONS.DECLINED && coverEndDate === UNDERWRITER_MANAGER_DECISIONS.DECLINED;
+  }
+  // Single amendment request
+  return value === UNDERWRITER_MANAGER_DECISIONS.DECLINED || coverEndDate === UNDERWRITER_MANAGER_DECISIONS.DECLINED;
+};
+
+/**
+ * Evaluated whether facility amendment is eligible
+ * for ACBS interaction based on myriads of conditions.
+ * This function evaluated across all amendment types.
+ * @param {Object} amendment Facility amendments object
+ */
+const canSendToAcbs = (amendment) => {
+  // Ensure at least one of the attribute has been amended
+  const hasBeenAmended = amendment.changeCoverEndDate || amendment.changeFacilityValue;
+  // Amendment status is marked as `Completed`
+  const completed = amendment.status === AMENDMENT_STATUS.COMPLETED;
+  // Amendment has been submitted by PIM team
+  const pim = Boolean(amendment.submittedByPim);
```

Comment:
I don't think this line is needed. `amendment.submittedByPim` is always boolean and we can use that directly in the conditions below

Reply:
I have removed Boolean but to keep the code clean I have left the variable assignment.